# Kazakh translation for update-manager
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2011-10-18 03:11+0000\n"
"PO-Revision-Date: 2011-09-16 09:52+0000\n"
"Last-Translator: jmb_kz <Unknown>\n"
"Language-Team: Kazakh <kk@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Launchpad-Export-Date: 2011-10-18 14:52+0000\n"
"X-Generator: Launchpad (build 14157)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:416 ../UpdateManager/Core/utils.py:416
msgid "0 kB"
msgstr "0 кБ"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:419 ../UpdateManager/Core/utils.py:419
msgid "1 kB"
msgstr "1 кБ"

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:422 ../UpdateManager/Core/utils.py:422
#, python-format
msgid "%.0f kB"
msgstr "%.0f кБ"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:425 ../UpdateManager/Core/utils.py:425
#, python-format
msgid "%.1f MB"
msgstr "%.1f Мб"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:210 ../DistUpgrade/distro.py:438
#, python-format
msgid "Server for %s"
msgstr "%s үшін сервер"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:228 ../DistUpgrade/distro.py:234
#: ../DistUpgrade/distro.py:250
msgid "Main server"
msgstr "Негізгі сервер"

#: ../DistUpgrade/distro.py:254
msgid "Custom servers"
msgstr "Өзіңіздің серверлеріңіз"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:141
msgid "Could not calculate sources.list entry"
msgstr "sources.list элементін санау мүмкін емес"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:244
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Пакеттер файлдарын табу мүмкін емес, мүмкін бұл Ubuntu дистрибутивының "
"дискісі да емес, немесе бұрыс сәулетті диск шығар?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:287
msgid "Failed to add the CD"
msgstr "CD дискі қосу сәтсіз аяқталды"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:288
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"CD дискіні қосу кезінде қате туындады, жаңарту тоқтатылатын болады. Егер бұл "
"дұрыс Ubuntu CD дискісі болған жағдайда, осы қате туралы хабарлауыңызды "
"сұраймыз.\n"
"\n"
"Қате туралы мәлімдеме:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:138
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Қате етіп белгіленген пакетті жою"

#: ../DistUpgrade/DistUpgradeCache.py:141
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"\"%s\" пакеті (немесе пакеттері) үйлесімді емес, оны (оларды) қайта орнату "
"қажет, бірақ та оның (олардың) архивті файл(дары) табылмады. Әрі қарай "
"жалғастыру үшін осы пакетті (пакеттерді) жойғыңыз келеді ме?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:239
msgid "The server may be overloaded"
msgstr "Мүмкін сервер асыра тиелген"

#: ../DistUpgrade/DistUpgradeCache.py:352
msgid "Broken packages"
msgstr "Бұзылған пакеттер"

#: ../DistUpgrade/DistUpgradeCache.py:353
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Жүйеде, осы бағдарламамен жөндеу мүмкін емес, бұзылған пакеттер бар. Әрі "
"қарай жалғастыру үшін оларды synaptic немесе apt-get бағдарламасының "
"біреуінің көмегімен жөндеңіз."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:648
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Жаңартуды есептеу кезінде шешілмейтін қате пайда болды:\n"
"%s\n"
"\n"
" Ол келесіден шығуы мүмкін:\n"
" * Ubuntu-ның пререлизді нұсқасына дейін жаңартылуыдан\n"
" * Ubuntu-ның ағымдағы пререлизді нұсқасын жүктелуінен\n"
" * Ubuntu қолдау көрсетпейтін ресми емес бағдарламалар пакеттерінен\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:658
msgid "This is most likely a transient problem, please try again later."
msgstr "Мүмкін бұл уақытша мәселе, кейінірек тағы байқап көріңіз."

#: ../DistUpgrade/DistUpgradeCache.py:661
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Егер ұсынылғанның шешімнің біреуі де көмектеспесе, осы қате туралы "
"терминалда \"ubuntu-bug update-manager\" командасын орындау арқылы "
"хабарлаңыз."

#: ../DistUpgrade/DistUpgradeCache.py:666
#: ../UpdateManager/UpdateManager.py:1023
msgid "Could not calculate the upgrade"
msgstr "Жүйенің жаңартуын есептеу мүмкін емес"

#: ../DistUpgrade/DistUpgradeCache.py:717
msgid "Error authenticating some packages"
msgstr "Кейбір пакеттердің растығын тексеру кезінде қате туындады"

#: ../DistUpgrade/DistUpgradeCache.py:718
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Кейбір пакеттердің шынайлығын тексеру мүмкін емес. Мүмкін бұл желімен қысқа "
"уақытты мәселе, кейінірек тағы қайталап көріңіз. Төменде шынайлығы тексеріле "
"алынбаған пакеттер тізімі көрсетілген."

#: ../DistUpgrade/DistUpgradeCache.py:738
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"\"%s\" пакеті жойылуға тиісті етіп белгіленген, бірақ ол жойылуға тыйым "
"салынған тізімде."

#: ../DistUpgrade/DistUpgradeCache.py:742
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "\"%s\" негізгі пакеті жойылуға тиісті етіп белгіленген."

#: ../DistUpgrade/DistUpgradeCache.py:751
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr ""
"Қара тізімге кірістірілген '%s' нұсқасының орнатылу мүмкіндігі орындалуда"

#: ../DistUpgrade/DistUpgradeCache.py:869
#, python-format
msgid "Can't install '%s'"
msgstr "\"%s\" орнату мүмкін емес"

#: ../DistUpgrade/DistUpgradeCache.py:870
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Сұранылған пакетті орнату мүмкін емес.  Осы қате туралы терминалда \"ubuntu-"
"bug update-manager\" командасын орындау арқылы хабарлаңыз."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:881
msgid "Can't guess meta-package"
msgstr "Мета-пакетті теріп алынбады"

#: ../DistUpgrade/DistUpgradeCache.py:882
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Сіздің жүйеңізде ubuntu-desktop, kubuntu-desktop, xubuntu-desktop немесе "
"edubuntu-desktop пакеттері орнатылмаған, сондықтан сізде қандай Ubuntu "
"жүйесі жүктелінгенін анықтау мүмкін емес.\n"
" Алдымен, жоғары аталған пакеттердің біреуін, synaptic немесе apt-get "
"бағдарламасының көмегімен орнатыңыз."

#: ../DistUpgrade/DistUpgradeController.py:100
msgid "Reading cache"
msgstr "Кэшті оқу"

#: ../DistUpgrade/DistUpgradeController.py:205
msgid "Unable to get exclusive lock"
msgstr "Ерекше бөгеттеу (блокировка) алу мүмкін емес"

#: ../DistUpgrade/DistUpgradeController.py:206
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Әдетте, бұл, пакеттерді басқару бағдарламаның (мысалыға, apt-get aptitude) "
"басқа көшірмесі жұмыс істегінін білдіреді. Әрі қарай осы бағдарламамен жұмыс "
"істеу үшін, басқа бағдарлама көшірмесін жабыңыз."

#: ../DistUpgrade/DistUpgradeController.py:239
msgid "Upgrading over remote connection not supported"
msgstr "Алыстаған қосылыста жаңартуға тыйым салынған"

#: ../DistUpgrade/DistUpgradeController.py:240
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Сіз жаңарту мүмкіндігі жоқ клиенттік бағдарлама көмегімен ssh-қосылыспенен "
"орындағыңыз келіп тұр. Текстік режимде \"do-release-upgrade\" қолданып "
"жаңартыңыз.\n"
"\n"
"Жаңарту тоқтатылды. Жаңартуды ssh қолданбай-ақ жасап көріңіз."

#: ../DistUpgrade/DistUpgradeController.py:254
msgid "Continue running under SSH?"
msgstr "SSH астында жұмысты жалғастыру қажет пе?"

#: ../DistUpgrade/DistUpgradeController.py:255
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Бұл сеанс ssh арқылы жүруде. Жаңарту барысы сәтсіз болып қалатын болса, "
"жүйені бұрыңғы қалыпқа келтіру қиынға соғады,\n"
"сондықтан жаңартуларды ssh арқылы жасаумауға ескертеміз.\n"
"\n"
"Егер жалғастырамын десеңіз, ssh қосымша қызметі \"%s\" портында жұмыс "
"істейтін болады.\n"
"Әрі қарай жалғастырасыз ба?"

#: ../DistUpgrade/DistUpgradeController.py:269
msgid "Starting additional sshd"
msgstr "Қосымша sshd қосылуы"

#: ../DistUpgrade/DistUpgradeController.py:270
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Қайта қалпына келтіруді жеңілдету үшін, қосымша sshd қызмет '%s' портында "
"жүктелінеді. Егер бірдеңе болып қалса, осы қызметке ssh көмегімен қосыла "
"аласыз.\n"

#: ../DistUpgrade/DistUpgradeController.py:278
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Егер сізде бранмауэр қосылған болса, сізге осы портты уақытша ашуыңыз қажет "
"болатын шығар. Бұл қауіпті ықтимал болғандықтан, порт автоматты ашылмайтын "
"болады. Сіз оны өзіңіз келедей ашуыңызға болады:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeController.py:349
#: ../DistUpgrade/DistUpgradeController.py:394
msgid "Can not upgrade"
msgstr "Жаңарту мүмкін емес"

#: ../DistUpgrade/DistUpgradeController.py:350
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "'%s', '%s' дейін осы қосымша құралмен мүмкін емес."

#: ../DistUpgrade/DistUpgradeController.py:359
msgid "Sandbox setup failed"
msgstr "\"Құмсалғыш\" (песочница) орнату сәтсіз"

#: ../DistUpgrade/DistUpgradeController.py:360
msgid "It was not possible to create the sandbox environment."
msgstr "Қауіпсіз орта (құмсалғыш (песочница)) құру мүмкін емес."

#: ../DistUpgrade/DistUpgradeController.py:366
msgid "Sandbox mode"
msgstr "\"Құмсалғыш\" (песочница) режимі (қауіпсіз орта)"

#: ../DistUpgrade/DistUpgradeController.py:367
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"Жаңарту қауіпсіз ортада (тестте) жүруде. Барлық өзгертулер '%s' жазылады "
"және де қайта жүктеуден кейін жойылатын болады.\n"
"\n"
"Жүйелік бумалардағы *ЕШҚАНДАЙ* өзгертулер қазіргі уақыттан, келесі қайта "
"жүктеуге дейін тұрақты болмайды."

#: ../DistUpgrade/DistUpgradeController.py:395
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Python дұрыс емес орнатылған. \"/usr/bin/python\" символдық сілтемені "
"жөндеңіз."

#: ../DistUpgrade/DistUpgradeController.py:421
msgid "Package 'debsig-verify' is installed"
msgstr "\"debsig-verify\" пакеті орнатылған"

#: ../DistUpgrade/DistUpgradeController.py:422
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Егер де осы пакет орнатылмаған болса, жаңарту жалғасу мүмкін емес.\n"
"Ең алдымен оны Synaptic немесе 'apt-get remove debsig-verify' арқылы "
"жойыңыз, кейін жаңартуды басынан жасап көріңіз."

#: ../DistUpgrade/DistUpgradeController.py:432
msgid "Include latest updates from the Internet?"
msgstr "Соңғы жаңартуларды Интернет желісінен жүктейін бе?"

#: ../DistUpgrade/DistUpgradeController.py:433
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Соңғы жаңартулар интернеттен жүктеліну мүмкін емес және жүйенің толық "
"жаңарту кезіндегі орнатыла алынбайды. Егер де сіз интернетке қосылған "
"болсаңыз, оны қатаң түрде ұсынамыз.\n"
"\n"
"Жүйенің жаңартылуы ұзақ уақыт үрдіс, одан кейін сіздің жүйеңіз актуальды "
"күйде болады. Сіз оны жасамауыңыз болады, бірақ та біз соңғы жаңартуларды "
"тезірек орнатуыңызға ұсынамыз.\n"
"Егер де сіз 'Жоқ' деп жауап берсеңіз, жаңартулар интернеттен жүктелінбейтін "
"болады."

#: ../DistUpgrade/DistUpgradeController.py:653
#, python-format
msgid "disabled on upgrade to %s"
msgstr "%s дейін жаңарту кезінде бөгеттелген"

#: ../DistUpgrade/DistUpgradeController.py:680
msgid "No valid mirror found"
msgstr "Дұрыс жұмыс істейтін айна табылмады"

#: ../DistUpgrade/DistUpgradeController.py:681
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Сіздің репозиторий туралы ақпаратты оқу кезінде, дистрибутивті жаңарту "
"айналары табылмады. Бұл ішкі айнаны қолдану немесе айналар туралы ақпарат "
"ескірген кезде болуы мүмкін.\n"
"\n"
"Сіз 'sources.list' қайта жазуыңызды қалайсыз ба? Егер 'Иә' таңдасаңыз, "
"барлық '%s' жазу '%s' дейін жаңартылатын болады.\n"
"Егер де 'Жоқ' таңдасаңыз, жаңарту тоқтатылады."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:701
msgid "Generate default sources?"
msgstr "Қалыпты қайнар көздерді генерациялайын ба?"

#: ../DistUpgrade/DistUpgradeController.py:702
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"'sources.list' файлындағы '%s' үшін жазу табылмады.\n"
"\n"
"'%s' үшін қалыпты жазуды қосу қажет пе? Егер 'Жоқ' таңдалғаны жаңартудан бас "
"тартқаныңыз."

#: ../DistUpgrade/DistUpgradeController.py:737
msgid "Repository information invalid"
msgstr "Репозиторий туралы ақпарат қате"

#: ../DistUpgrade/DistUpgradeController.py:738
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug using the command 'ubuntu-bug update-manager' in a "
"terminal."
msgstr ""
"Репозиторий жайлы мәліметті жаңарту бұрыс файл құрылуына әкеліп соқтырды. "
"Осы қате туралы терминалда \"ubuntu-bug update-manager\" командасын орындау "
"арқылы хабарлаңыз."

#: ../DistUpgrade/DistUpgradeController.py:746
msgid "Third party sources disabled"
msgstr "Үшінші жақты бағдарламалар қайнар көзі өшірілді"

#: ../DistUpgrade/DistUpgradeController.py:747
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"'sources.list' файлындағы қайнарлары қарастырылмайтын етіп белгіленген. "
"Кейінірек сіз оларды қайтадан «Бағдарламалар қайнар көздер» қосымша "
"бағдарламасында немесе сіздің пакеттер менеджерінде оларды оң етіп белгілеу "
"арқылы қолданылуыңыз мүмкін."

#: ../DistUpgrade/DistUpgradeController.py:787
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Пакет(тер) тұрақсыз күйде"

#: ../DistUpgrade/DistUpgradeController.py:790
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"жаңартылатын \"%s\" пакеті толық емес және қайта орнатылуға тиіс, бірақ оған "
"қатысты мұрағат файлы (архивный файл) табылмады. Бұл пакетті жекешелей "
"жаңартыңыз немесе жойыңыз."

#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Error during update"
msgstr "Жаңарту кезінде қате туындады"

#: ../DistUpgrade/DistUpgradeController.py:839
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Жаңарту кезіндегі мәселе туындады. Әдетте бұл желідегі мәселелерінің "
"кесірінен болуы мүмкін. Желілік қосылыстарды тексеріп, қайталап көріңіз."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:848
#: ../UpdateManager/UpdateManager.py:747
msgid "Not enough free disk space"
msgstr "Дискіде қажетті бос орын жоқ"

#: ../DistUpgrade/DistUpgradeController.py:849
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Жаңарту үзілді. Жаңарту үшін \"%s\" дискісінде %s бос орын қажет. Ең аз "
"дегенде \"%s\" дискісінде қосымша %s орын босатыңыз. Қоқыс шелегін босатыңыз "
"және \"sudo apt-get clean\" команда орындау арқылы, алдыңғы орнатулардың "
"уақытша жүктелген пакеттерін жойыңыз."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:878
#: ../DistUpgrade/DistUpgradeController.py:1630
msgid "Calculating the changes"
msgstr "Өзгертулерді есептеу"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:910
msgid "Do you want to start the upgrade?"
msgstr "Жүйенің жаңартуын бастау қажет пе?"

#: ../DistUpgrade/DistUpgradeController.py:976
msgid "Upgrade canceled"
msgstr "Жаңарту болдырылмады"

#: ../DistUpgrade/DistUpgradeController.py:977
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Бұл жаңарту тоқтатылып, жүйе бастапқы күйге келтірілетін болады. Жаңартуды "
"кейінірек жалғастыруыңызға болады."

#: ../DistUpgrade/DistUpgradeController.py:983
#: ../DistUpgrade/DistUpgradeController.py:1109
msgid "Could not download the upgrades"
msgstr "Жаңартуларды жүктеу мүмкін емес"

#: ../DistUpgrade/DistUpgradeController.py:984
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Жаңарту үзілді. Интернетпен қосылысыңызды не орнатушы дискісін тексеріңіз "
"де, жаңартуды қайтадан орындап көріңіз. Барлық жүктелген файлдар "
"сақталынатын болады."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1060
#: ../DistUpgrade/DistUpgradeController.py:1097
#: ../DistUpgrade/DistUpgradeController.py:1202
msgid "Error during commit"
msgstr "Іске асыру кезінде қате туындады"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1062
#: ../DistUpgrade/DistUpgradeController.py:1099
#: ../DistUpgrade/DistUpgradeController.py:1241
msgid "Restoring original system state"
msgstr "Жүйені бастапқы қалпына келтіру"

#: ../DistUpgrade/DistUpgradeController.py:1063
#: ../DistUpgrade/DistUpgradeController.py:1078
#: ../DistUpgrade/DistUpgradeController.py:1100
msgid "Could not install the upgrades"
msgstr "Жаңартуларды орнату мүмкін емес"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1068
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Жаңарту үзілді. Сіздің жүйеңіз бұзылып, қолдануға жарамсыз болуы мүмкін. "
"Қазір қайта қалпына келтіру үрдісі орындалатын болады (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1073
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug using the command 'ubuntu-bug update-manager' in a "
"terminal and include the files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Осы қате туралы терминалда \"ubuntu-bug update-manager\" командасын орындау "
"арқылы хабарлаңыз, хабарламаға қоса /var/log/dist-upgrade/ бумасындағы "
"файлдарды тиеңіз.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1110
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Жаңарту үзілді. Интернетпен қосылысыңызды не орнатушы дискісін тексеріңіз "
"де, жаңартуды қайтадан орындап көріңіз. "

#: ../DistUpgrade/DistUpgradeController.py:1190
msgid "Remove obsolete packages?"
msgstr "Ескірген пакеттерді жоя қажет пе?"

#: ../DistUpgrade/DistUpgradeController.py:1191
#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Keep"
msgstr "_Қалдыру"

#: ../DistUpgrade/DistUpgradeController.py:1191
msgid "_Remove"
msgstr "_Жою"

#: ../DistUpgrade/DistUpgradeController.py:1203
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Тазарту кезінді, мәселе туындады. Толық ақпарат төменде сипатталған. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1279
msgid "Required depends is not installed"
msgstr "Қажетті тәуелділіктер орнатылмаған"

#: ../DistUpgrade/DistUpgradeController.py:1280
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Қажетті \"%s\" тәуелділігі орнатылмаған. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1527
#: ../DistUpgrade/DistUpgradeController.py:1597
msgid "Checking package manager"
msgstr "Пакеттер менеджері тексеру"

#: ../DistUpgrade/DistUpgradeController.py:1532
msgid "Preparing the upgrade failed"
msgstr "Жаңартуға дайындық сәтсіз аяқталды"

#: ../DistUpgrade/DistUpgradeController.py:1533
msgid ""
"Preparing the system for the upgrade failed. Please report this using the "
"command 'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""
"Жүйені жаңартуға дайындық сәтсіз аяқталды. Осы қате туралы терминалда "
"\"ubuntu-bug update-manager\" командасын орындау арқылы хабарлаңыз, "
"хабарламаға қоса /var/log/dist-upgrade/ бумасындағы файлдарды тиеңіз."

#: ../DistUpgrade/DistUpgradeController.py:1549
msgid "Getting upgrade prerequisites failed"
msgstr "Жаңартуға дайындық сәтсіз аяқталды"

#: ../DistUpgrade/DistUpgradeController.py:1550
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug using the command 'ubuntu-bug update-manager' in "
"a terminal and include the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""
"Жаңартуды орындау үшін жүйе қажетті талаптар тізімін ала алмай жатыр. "
"Жаңарту үзілетін болады да, жүйе бастапқы күйге оралады.\n"
"\n"
"Осы қате туралы терминалда \"ubuntu-bug update-manager\" командасын орындау "
"арқылы хабарлаңыз, хабарламаға қоса /var/log/dist-upgrade/ бумасындағы "
"файлдарды тиеңіз."

#: ../DistUpgrade/DistUpgradeController.py:1581
msgid "Updating repository information"
msgstr "Репозиторий туралы ақпаратын жаңарту"

#: ../DistUpgrade/DistUpgradeController.py:1588
msgid "Failed to add the cdrom"
msgstr "Дискті қосу сәтсіз аяқталды"

#: ../DistUpgrade/DistUpgradeController.py:1589
msgid "Sorry, adding the cdrom was not successful."
msgstr "Кешіріңіз, дискті қосу сәтсіз аяқталды."

#: ../DistUpgrade/DistUpgradeController.py:1617
msgid "Invalid package information"
msgstr "Пакет туралы дұрыс емес ақпарат"

#: ../DistUpgrade/DistUpgradeController.py:1618
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal and include the files in "
"/var/log/dist-upgrade/ in the bug report."
msgstr ""
"Пакет жайлы мәлімет жаңартуынан кейін, \"%s\" негізгі пакет ендігәрі "
"табылмады.\n"
"Бұл ірі қате жайлы мәлімдейді, осы қате туралы терминалда \"ubuntu-bug "
"update-manager\" командасын орындау арқылы хабарлаңыз, хабарламаға қоса "
"/var/log/dist-upgrade/ бумасындағы файлдарды тиеңіз."

#: ../DistUpgrade/DistUpgradeController.py:1636
#: ../DistUpgrade/DistUpgradeController.py:1689
msgid "Fetching"
msgstr "Жүктелу"

#: ../DistUpgrade/DistUpgradeController.py:1642
#: ../DistUpgrade/DistUpgradeController.py:1693
msgid "Upgrading"
msgstr "Жаңартылу"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1647
#: ../DistUpgrade/DistUpgradeController.py:1695
#: ../DistUpgrade/DistUpgradeController.py:1702
#: ../DistUpgrade/DistUpgradeController.py:1713
msgid "Upgrade complete"
msgstr "Жаңарту аяқталды"

#: ../DistUpgrade/DistUpgradeController.py:1648
#: ../DistUpgrade/DistUpgradeController.py:1696
#: ../DistUpgrade/DistUpgradeController.py:1703
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "Жаңарту аяқталды, бірақ жаңарту кезінде қателер пайда болды."

#: ../DistUpgrade/DistUpgradeController.py:1655
msgid "Searching for obsolete software"
msgstr "Ескерген бағдарламаларды іздеу"

#: ../DistUpgrade/DistUpgradeController.py:1664
msgid "System upgrade is complete."
msgstr "Жүйенің жаңартылуы аяқталды."

#: ../DistUpgrade/DistUpgradeController.py:1714
msgid "The partial upgrade was completed."
msgstr "Жарым-жартылай жаңарту аяқталды."

#: ../DistUpgrade/DistUpgradeQuirks.py:201
msgid "evms in use"
msgstr "evms қолданыста"

#: ../DistUpgrade/DistUpgradeQuirks.py:202
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Жүйеңіз /proc/mounts ішінде \"evms\" дискілік бөлім менеджерін қолданып "
"жатыр. Ендігәрі \"evms\" бағдарламалық қамтамасыздандыруға қолдау "
"көрсетілмейді. Оны жауып жаңартуды қайтадан жасап көріңіз."

#: ../DistUpgrade/DistUpgradeQuirks.py:506
msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:508
msgid ""
"The support in Ubuntu 11.04 for your intel graphics hardware is limited and "
"you may encounter problems after the upgrade. Do you want to continue with "
"the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:528
#: ../DistUpgrade/DistUpgradeQuirks.py:556
#: ../DistUpgrade/DistUpgradeQuirks.py:583
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Жаңарту, жұмыс үстелінің эффектерінің сапасының, графикамен белсенді жұмыс "
"істейтін ойындар мен бағдарламалардың өнімділігін төмендеуіне әкеліп "
"соқтыруы мүмкін."

#: ../DistUpgrade/DistUpgradeQuirks.py:532
#: ../DistUpgrade/DistUpgradeQuirks.py:560
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Бұл жүйеде NVIDIA \"nvidia\" драйвері қолданылуда. Ubuntu 10.04 LTS сіздің "
"бейнекартаңызбен жұмыс істейтін осындай драйверлер нұсқалары жетімсіз.\n"
"\n"
"Жалғастыру қажет пе?"

#: ../DistUpgrade/DistUpgradeQuirks.py:587
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Бұл жүйеде AMD \"fglrx\" драйвері қолданылуда. Ubuntu 10.04 LTS сіздің "
"бейнекартаңызбен жұмыс істейтін осындай драйверлер нұсқалары жетімсіз.\n"
"\n"
"Жалғастыру қажет пе?"

#: ../DistUpgrade/DistUpgradeQuirks.py:617
msgid "No i686 CPU"
msgstr "i686 сәулетімен үйлесімді орталық процессор жоқ"

#: ../DistUpgrade/DistUpgradeQuirks.py:618
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Сіздің жүйеңізде \"cmov\" кеңейтуі жоқ немесе i586 сәулетімен үйлесімді "
"орталық процессор қолданылып тұр. Пакеттердің барлығы ең аз дегенде i686 "
"сәулетімен үйлесімді орталық процессорлармен тиімді жұмыс істеу үшін етіп "
"жиналған."

#: ../DistUpgrade/DistUpgradeQuirks.py:654
msgid "No ARMv6 CPU"
msgstr "ARMv6 орталық процессоры (CPU) жоқ"

#: ../DistUpgrade/DistUpgradeQuirks.py:655
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Сіздің жүйеңіз ARMv6 сәулетінен төмен, ARM сәулет процессорын қолданып "
"жатыр. karmic ішіндегі барлық пакеттер ARMv6 сәулетінен төмен емес "
"сәулеттерге арналып қалыпқа келтірілген. Ағымдағы құралдар жабдықтармен, "
"жүйеңізді Ubuntu жаңа релизіне дейін жаңарту мүмкін емес."

#: ../DistUpgrade/DistUpgradeQuirks.py:675
msgid "No init available"
msgstr "init қызметі жетімсіз"

#: ../DistUpgrade/DistUpgradeQuirks.py:676
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Сіздің жүйеңіз init қызметінсіз, мысалыға Linux-VServer жоқ, "
"виртуализацияланған орта секілді. Ubuntu 10.04 LTS осындай түрлі ортада "
"жұмыс істей алмайды, қажет болса, сіз өзіңіздің виртуалды машинаңыздың "
"баптауларын сәйкесінше қалыпқа келтіруіңіз қажет.\n"
"\n"
"Жалғастыруды қалайсыз ба?"

#: ../DistUpgrade/DistUpgradeMain.py:45
msgid "Sandbox upgrade using aufs"
msgstr "aufs қолданып, қауіпсіз ортада (құмсалғышта) жаңарту"

#: ../DistUpgrade/DistUpgradeMain.py:47
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "Жаңартулар пакеттері бар CD/DVD дискілерін көрсетілген жерден іздеу"

#: ../DistUpgrade/DistUpgradeMain.py:53
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Интерфейсті қолдану. Қазір жетімділер: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:56
msgid "*DEPRECATED* this option will be ignored"
msgstr "Бұл параметр ЕСКЕРДІ және қолданылмайтын болады"

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Тек жартылай жаңартуларды дайындау (sources.list үстінен басылмайтын болады)"

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid "Disable GNU screen support"
msgstr "GNU экранның қолдауын алып тастау"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid "Set datadir"
msgstr "Мәліметтер бумасын орнату"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:189
#: ../UpdateManager/DistUpgradeFetcherKDE.py:147
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "\"%s\" приводына \"%s\" салыңыз"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
msgid "Fetching is complete"
msgstr "Жүктеу аяқталды"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:216
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "%li файлдың %li, %sбайт/сек жылдамдығымен жүктелінді"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:295
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:301
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:361
#, python-format
msgid "About %s remaining"
msgstr "Шамамен %s қалды"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
#, python-format
msgid "Fetching file %li of %li"
msgstr "%li ішінен %li файл жүктелуде"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:253
msgid "Applying changes"
msgstr "Өзгерістерді қолдану"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:211
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "dependency problems - leaving unconfigured"
msgstr "тәуелділік қателері - қалыпқа келтірмеген етіп қалдыру"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:216
#: ../DistUpgrade/DistUpgradeViewKDE.py:268
#, python-format
msgid "Could not install '%s'"
msgstr "\"%s\" орнату мүмкін емес"

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Жаңарту әрі қарай жалғасатын болады, бірақта \"%s\" пакет(тер) жұмыс "
"істемейтін күйге келтірілуі мүмкін. Осы қате жайлы хабарлау туралы "
"ойластыруыңызды сұраймыз."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:234
#: ../DistUpgrade/DistUpgradeViewKDE.py:290
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"\"%s\"\n"
"өзгертілген баптау файлды үстінен басып сақтайын ба?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Егер баптау файлдың бұрыңғы нұсқасын жаңа нұсқасымен алмастытатын болсаңыз, "
"онда ол баптау файлдағы өзгертулеріңіздің барлығы жоғалады."

#: ../DistUpgrade/DistUpgradeViewGtk.py:250
#: ../DistUpgrade/DistUpgradeViewGtk3.py:253
#: ../DistUpgrade/DistUpgradeViewKDE.py:313
msgid "The 'diff' command was not found"
msgstr "\"diff\" командасы табылмады"

#: ../DistUpgrade/DistUpgradeViewGtk.py:463
#: ../DistUpgrade/DistUpgradeViewGtk3.py:469
#: ../DistUpgrade/DistUpgradeViewText.py:90
msgid "A fatal error occurred"
msgstr "Аса ауыр қате туындады"

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:470
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Осы жайлы қате ретінде хабарлаңыз (егер де түк жасамаған болсаңыз) да, "
"/var/log/dist-upgrade/main.log мен /var/log/dist-upgrade/apt.log файлдарды "
"есепке тиеп қойыңыз. Жаңарту үзілді.\n"
"sources.list файлыңыздың көшірмесі осы жерде сақталынған "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:481
#: ../DistUpgrade/DistUpgradeViewGtk3.py:487
msgid "Ctrl-c pressed"
msgstr "Ctrl-c басылды"

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:488
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Операция тоқтатылатын болады, жүйеңіз жұмысқа жарамайтын болып қалуы мүмкін. "
"Жалғастырасыз ба?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:630
#: ../DistUpgrade/DistUpgradeViewGtk3.py:621
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Мәліметтерді жоғалтуын болдырмау үшін, барлық ашық бағдарламалар мен "
"құжаттарды жабыңыз."

#: ../DistUpgrade/DistUpgradeViewGtk.py:644
#: ../DistUpgrade/DistUpgradeViewGtk3.py:635
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Ендігәрі Canonical (%s) компаниясымен қолдау көрсетілмейді"

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:636
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Ескірген нұсқаны орнату (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:637
#, python-format
msgid "Remove (%s)"
msgstr "Жою (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:638
#, python-format
msgid "No longer needed (%s)"
msgstr "Ендігәрі қажет емес (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:639
#, python-format
msgid "Install (%s)"
msgstr "Орнату (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:640
#, python-format
msgid "Upgrade (%s)"
msgstr "Жаңарту (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:149
msgid "Media Change"
msgstr "Сақтауышты ауыстыру"

#: ../DistUpgrade/DistUpgradeViewKDE.py:325
msgid "Show Difference >>>"
msgstr "Айырмашылықтарды көрсету >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:328
msgid "<<< Hide Difference"
msgstr "<<< Айырмашылықтарды жасыру"

#: ../DistUpgrade/DistUpgradeViewKDE.py:543
msgid "Error"
msgstr "Қате"

#: ../DistUpgrade/DistUpgradeViewKDE.py:555
msgid "&Cancel"
msgstr "_Болдырмау"

#: ../DistUpgrade/DistUpgradeViewKDE.py:557
#: ../DistUpgrade/DistUpgradeViewKDE.py:799
msgid "&Close"
msgstr "&Жабу"

#: ../DistUpgrade/DistUpgradeViewKDE.py:603
msgid "Show Terminal >>>"
msgstr "Терминалды көрсету >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:606
msgid "<<< Hide Terminal"
msgstr "<<< Терминалды жасыру"

#: ../DistUpgrade/DistUpgradeViewKDE.py:686
msgid "Information"
msgstr "Мәлімет"

#: ../DistUpgrade/DistUpgradeViewKDE.py:737
#: ../DistUpgrade/DistUpgradeViewKDE.py:782
#: ../DistUpgrade/DistUpgradeViewKDE.py:785 ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Details"
msgstr "Көбірек білу"

#: ../DistUpgrade/DistUpgradeViewKDE.py:763
#, python-format
msgid "No longer supported %s"
msgstr "Қолдау көрсетілмейді (%s)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:765
#, python-format
msgid "Remove %s"
msgstr "%s жою"

#: ../DistUpgrade/DistUpgradeViewKDE.py:767
#: ../DistUpgrade/DistUpgradeViewText.py:179
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "%s жою (автоматты орнатылған)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:769
#, python-format
msgid "Install %s"
msgstr "%s орнату"

#: ../DistUpgrade/DistUpgradeViewKDE.py:771
#, python-format
msgid "Upgrade %s"
msgstr "%s жаңарту"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
#: ../DistUpgrade/DistUpgradeViewText.py:227
msgid "Restart required"
msgstr "Компьютерді қайта жүктеу қажет"

#: ../DistUpgrade/DistUpgradeViewKDE.py:795
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Жаңартуды аяқтау үшін компьютерді қайта жүктеңіз</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:798 ../DistUpgrade/DistUpgrade.ui.h:25
#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid "_Restart Now"
msgstr "_Қазір қайта жүктеу"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:816
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Жаңартулды үзесіз бе?</big></b>\n"
"Егер де сіз жаңартуды үзетін болсаңыз, жүйе тұрақсыз жұмыс істеуі мүмкін. "
"Жаңартуды әрі қарай жалғастыруды қадала жасауға ұсынамыз."

#: ../DistUpgrade/DistUpgradeViewKDE.py:820
msgid "Cancel Upgrade?"
msgstr "Жаңартуды болдырмау қажет пе?"

#: ../DistUpgrade/DistUpgradeView.py:59
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li күн"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li сағат"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li минут"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li секунд"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:80
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:98
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:147
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Жүктеу, қосылыстың 1Мбит DSL жылдамдығында шамамен %s және модемдік 56Кбит "
"Dial-up жылдамдығында шамамен %s алатын болады."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Сіздің қосылысыңызда жүктеу шамамен %s алатын болады. "

#: ../DistUpgrade/DistUpgradeView.py:253 ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Preparing to upgrade"
msgstr "Жаңартуға дайындалу"

#: ../DistUpgrade/DistUpgradeView.py:254
msgid "Getting new software channels"
msgstr "Бағдарламалар қайнар көздерін өзгерту"

#: ../DistUpgrade/DistUpgradeView.py:255 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Getting new packages"
msgstr "Жаңа пакеттерді алу"

#: ../DistUpgrade/DistUpgradeView.py:256 ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Installing the upgrades"
msgstr "Жаңартуларды орнату"

#: ../DistUpgrade/DistUpgradeView.py:257 ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Cleaning up"
msgstr "Тазарту"

#: ../DistUpgrade/DistUpgradeView.py:342
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"Ендігәрі %(amount)d пакетке/пакеттерге Canonical қолдау көрсетпейді. Бірақ "
"та сіз қоғамдастықтың қолдауын ала аласыз."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:351
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d жойылатын болады."

#: ../DistUpgrade/DistUpgradeView.py:356
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d пакеті(тері) орнатылатын болады."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d пакеті(тері) жаңартылатын болады."

#: ../DistUpgrade/DistUpgradeView.py:367
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Жалпы %s жүктеу қажет. "

#: ../DistUpgrade/DistUpgradeView.py:371
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"Жаңартуларды жүктеу және орнату бірнеше сағат алуы мүмкін. Жүктеуден кейін "
"бұл процессті кері қайтаруға мүмкін болмайды."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:377 ../UpdateManager/UpdateManager.py:664
msgid "Your system is up-to-date"
msgstr "Сіздің жүйеңіз жаңартуларды қажет етпейді"

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Сіздің жүйеңізге ешбір жаңарту жетімді емес. Жаңарту кері қайтатын болады."

#: ../DistUpgrade/DistUpgradeView.py:391
msgid "Reboot required"
msgstr "Компьютерді қайта жүктеу қажет"

#: ../DistUpgrade/DistUpgradeView.py:392
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Жаңарту аяқталды және қайта жүктеу қажет. Қазір қайта жүктеу орындайсыз ба?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid "Could not run the upgrade tool"
msgstr "Жаңарту құралын ашу мүмкін емес"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:132
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:132
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""
"Жаңарту құралындаға бар секілді. Осы қате туралы терминалда \"ubuntu-bug "
"update-manager\" командасын орындау арқылы хабарлаңыз."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:207
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:207
msgid "Upgrade tool signature"
msgstr "Жаңарту құралын баспасы"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:214
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:214
msgid "Upgrade tool"
msgstr "Жаңарту құралы"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Failed to fetch"
msgstr "Алу мүмкін емес"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Жаңартуларды алу мүмкін сәтсіз аяқталды. Желілік мәселелер болуы мүмкін. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:253
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:253
msgid "Failed to extract"
msgstr "Шығарып алу (извлечь) сәтсіз"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Жаңартуларды шығарып алу сәтсіз аяқталды. Желімен не сервермен мәселелер "
"болуы мүмкін. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:259
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:259
msgid "Verification failed"
msgstr "Тексеріліс сәтсіз"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:260
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:260
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Тексеріліс сәтсіз аяқталды. Желімен не сервермен мәселелер болуы мүмкін. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:265
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:265
msgid "Authentication failed"
msgstr "Аутентификация сәтсіз"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:266
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:266
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"жаңартулардың шынайлығын растау сәтсіз аяқталды. Желімен не сервермен "
"мәселелер болуы мүмкін. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid "Can not run the upgrade"
msgstr "Жаңарту процессін ашу мүмкін емес"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:281
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:281
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Бұл әдетте /tmp нүктесі noexec параметрімен тіркелген жүйелерде пайда "
"болады. Нүктені noexec параметрсіз қайта тіркеңіз де, қайтадан жаңартып "
"көріңіз."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:287
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:287
#, python-format
msgid "The error message is '%s'."
msgstr "Қате туралы хабарлама \"%s\"."

#: ../DistUpgrade/DistUpgradeViewText.py:91
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Осы жайлы қате ретінде хабарлаңыз да, /var/log/dist-upgrade/main.log мен "
"/var/log/dist-upgrade/apt.log файлдарды есепке тиеп қойыңыз. Жаңарту "
"үзілді.\n"
"sources.list файлыңыздың көшірмесі осы жерде сақталынған "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:115
msgid "Aborting"
msgstr "Тоқтатып жатырмын"

#: ../DistUpgrade/DistUpgradeViewText.py:120
msgid "Demoted:\n"
msgstr "Төмендетілген:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:127
msgid "To continue please press [ENTER]"
msgstr "Жалғастыру үшін [ENTER] пернесін басыңыз"

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
#: ../DistUpgrade/DistUpgradeViewText.py:200
msgid "Continue [yN] "
msgstr "Жалғастыру [иЖ] "

#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:193
msgid "Details [d]"
msgstr "Толық ақпарат [т]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:159
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "y"
msgstr "и"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "n"
msgstr "ж"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "d"
msgstr "т"

#: ../DistUpgrade/DistUpgradeViewText.py:170
#, python-format
msgid "No longer supported: %s\n"
msgstr "Қолдау көрсетілмейді: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:175
#, python-format
msgid "Remove: %s\n"
msgstr "Жоюға: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:185
#, python-format
msgid "Install: %s\n"
msgstr "Орнатуға: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:190
#, python-format
msgid "Upgrade: %s\n"
msgstr "Жаңартуға: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:207
msgid "Continue [Yn] "
msgstr "Жалғастыру [Иж] "

#: ../DistUpgrade/DistUpgradeViewText.py:228
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Жаңартуды аяқтау үшін, компьютерді қайта жүктеу қажет.\n"
"Егер сіз \"и\" таңдасаңыз, жүйе қайта жүктелетін болады."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Ағымдағы жаңартуды үзу қажет пе?</big></b>\n"
"\n"
"Егер жаңартуды үзетін болсаңыз, жүйеңіз жұмыс істемейтін күйге айналуы "
"мүмкін. Жаңартуды қадала жалғастыруды сұраймыз."

#: ../DistUpgrade/DistUpgrade.ui.h:5
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Жаңартуды аяқтау үшін жүйеңізді қайта жүктеңіз.</big></b>\n"
"\n"
"Жалғастыру алдында өзіңіздің құжаттарыңызды сақтаңыз."

#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Жаңартуды бастайын ба?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "<b><big>Upgrading Ubuntu to version 11.10</big></b>"
msgstr "<b><big>Ubuntu-ны 11.10 нұсқасына дейін жаңарту</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Difference between the files"
msgstr "Файлдар арасындағы айырмашылықтар"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Distribution Upgrade"
msgstr "Дистрибутивті жаңарту"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Restarting the computer"
msgstr "Компьютер қайта жүктелуде"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Setting new software channels"
msgstr "Бағдарламалардың жаңа қайнар көздерін қосу"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "Terminal"
msgstr "Терминал"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Cancel Upgrade"
msgstr "Жаңартуды _болдырмау"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Continue"
msgstr "_Жалғастыру"

#: ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Replace"
msgstr "_Алмастыру"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Report Bug"
msgstr "_Қате туралы есепті жіберу"

#: ../DistUpgrade/DistUpgrade.ui.h:26
msgid "_Resume Upgrade"
msgstr "Жаңартуды _жалғастыру"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "_Start Upgrade"
msgstr "Жаңартуды _бастау"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:55
msgid "Please wait, this can take some time."
msgstr "Күтіңіз, бұл біраз уақыт алуы мүмкін."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:57
msgid "Update is complete"
msgstr "Жаңарту аяқталды"

#: ../UpdateManager/DistUpgradeFetcher.py:108
#: ../UpdateManager/DistUpgradeFetcherKDE.py:102
msgid "Could not find the release notes"
msgstr "Релиз туралы мәліметтері табылмады"

#: ../UpdateManager/DistUpgradeFetcher.py:109
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "The server may be overloaded. "
msgstr "Мүмкін сервер аса жүктеулі. "

#: ../UpdateManager/DistUpgradeFetcher.py:119
#: ../UpdateManager/DistUpgradeFetcherKDE.py:107
msgid "Could not download the release notes"
msgstr "Релиз туралы мәліметтері жүктеу мүмкін емес"

#: ../UpdateManager/DistUpgradeFetcher.py:120
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Please check your internet connection."
msgstr "Өзіңіздің интернетпен қосылысыңызды тексеріңіз."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:61
#: ../UpdateManager/DistUpgradeFetcherKDE.py:84
msgid "Upgrade"
msgstr "Жаңарту"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:88
#: ../data/gtkbuilder/UpdateManager.ui.h:13
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Release Notes"
msgstr "Релиз туралы мәлімет"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:127
#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
msgid "Downloading additional package files..."
msgstr "Пакеттердің қосымша файлдары жүктелуде..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:140
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "%sб/с жылдамдықпен %s файлдың %s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:142
#, python-format
msgid "File %s of %s"
msgstr "%s файлдың %s"

#: ../UpdateManager/ChangelogViewer.py:73
msgid "Open Link in Browser"
msgstr "Сілтемені веб шолғышта ашу"

#: ../UpdateManager/ChangelogViewer.py:76
msgid "Copy Link to Clipboard"
msgstr "Сілтемені алмастыру буферіне көшіру"

#: ../UpdateManager/GtkProgress.py:159
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "%(speed)s/с жылдамдықпен %(total)li файлдың %(current)li жүктелінуде"

#: ../UpdateManager/GtkProgress.py:164
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "%(total)li файлдың %(current)li жүктелінуде"

#: ../UpdateManager/UpdateManager.py:105 ../do-release-upgrade:90
msgid "Your Ubuntu release is not supported anymore."
msgstr "Сіздің Ubuntu нұсқаңызға қолдау ендігәрі көрсетілмейді."

#: ../UpdateManager/UpdateManager.py:106
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu Linux."
msgstr ""
"Ендігәрі сіз қауіпсіздік немесе маңызды жаңартуларды ала алмайтын боласыз. "
"Ubuntu соңғы релиздеріне дейін жаңартылыңыз."

#: ../UpdateManager/UpdateManager.py:114
msgid "Upgrade information"
msgstr "Жаңарту жайлы ақпарат"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:389
#, python-format
msgid "Version %s: \n"
msgstr "%s нұсқасы: \n"

#: ../UpdateManager/UpdateManager.py:445
msgid ""
"No network connection detected, you can not download changelog information."
msgstr "Желілік қосылыс жоқ, өзгертулер тізімін жүктеп алу мүмкін емес."

#: ../UpdateManager/UpdateManager.py:453
msgid "Downloading list of changes..."
msgstr "Өзгертулер тізімін жүктеу..."

#: ../UpdateManager/UpdateManager.py:497
msgid "_Deselect All"
msgstr "_Белгілердің барлығын алып тастау"

#: ../UpdateManager/UpdateManager.py:503
msgid "Select _All"
msgstr "Б_арлығын таңдау"

#: ../UpdateManager/UpdateManager.py:552
#, python-format
msgid "%(count)s update has been selected."
msgid_plural "%(count)s updates have been selected."
msgstr[0] "%(count)s жаңарту белгіленген."

#: ../UpdateManager/UpdateManager.py:556
#, python-format
msgid "%s will be downloaded."
msgstr "%s жүктелінетін болады."

#: ../UpdateManager/UpdateManager.py:568
msgid "The update has already been downloaded, but not installed"
msgid_plural "The updates have already been downloaded, but not installed"
msgstr[0] "Жаңарту(лар) жүктелінді, бірақ орнатылмады"

#: ../UpdateManager/UpdateManager.py:573
msgid "There are no updates to install"
msgstr "Орнатылатын жаңарту жоқ"

#. TRANSLATORS: this allows to switch the order of the count of
#. updates and the download size string (if needed)
#: ../UpdateManager/UpdateManager.py:579
#, python-format
msgid "%(count_str)s %(download_str)s"
msgstr "%(count_str)s %(download_str)s"

#: ../UpdateManager/UpdateManager.py:586
msgid "Unknown download size."
msgstr "Жүктелінетін көлем белгісіз."

#: ../UpdateManager/UpdateManager.py:612
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""
"Пакет жайлы мәлімет қашан жаңартылғаны белгісіз. Мәліметті жаңарту үшін "
"\"Тексеру\" батырмасын басыңыз."

#: ../UpdateManager/UpdateManager.py:618
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Пакеттер жайлы мәлімет %(days_ago)s күн бұрын жаңартылған.\n"
"Жаңартулардың жетімдігін тексеру үшін \"Тексеру\" батырмасын басыңыз."

#: ../UpdateManager/UpdateManager.py:623
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural ""
"The package information was last updated %(days_ago)s days ago."
msgstr[0] "Пакеттер жайлы мәлімет %(days_ago)s күн бұрын жаңартылған."

#: ../UpdateManager/UpdateManager.py:627
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] "Пакеттер жайлы мәлімет %(hours_ago)s сағат бұрын жаңартылған."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:632 ../UpdateManager/UpdateManager.py:634
#: ../UpdateManager/UpdateManager.py:636
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr "Пакеттер жайлы мәлімет %s минут бұрын жаңартылған."

#: ../UpdateManager/UpdateManager.py:638
msgid "The package information was just updated."
msgstr "Пакеттер жайлы мәлімет қазір ғана жаңартылды."

#: ../UpdateManager/UpdateManager.py:658
#: ../data/gtkbuilder/UpdateManager.ui.h:22
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Бағдарламалардың жаңартулары қателер мен осалдықтарды (уязвимости) жөндеп, "
"жаңа мүмкіншіліктерді қосады."

#: ../UpdateManager/UpdateManager.py:677
msgid "Software updates may be available for your computer."
msgstr "Сіздің компьютеріңіз үшін жаңартулар жетімді болуы мүмкін."

#: ../UpdateManager/UpdateManager.py:685
msgid "Welcome to Ubuntu"
msgstr "Ubuntu-ға қош келдіңіз"

#: ../UpdateManager/UpdateManager.py:687
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from Applications later."
msgstr ""
"Осы бағдарламалық қамтамасыздандырудың жаңартулары осы Ubuntu нұсқасының "
"шығарылымынан кейін шығарылды. Егер де оларды қазір орнатуыңыз келіп тұрған "
"жоқ болса, кейінірек Бағдарламалар мәзірінен \"Жаңартулар менеджерін\" "
"қолданып орната аласыз."

#: ../UpdateManager/UpdateManager.py:689
msgid ""
"These software updates have been issued since this version of Ubuntu was "
"released. If you don't want to install them now, choose \"Update Manager\" "
"from the Administration Menu later."
msgstr ""
"Осы бағдарламалық қамтамасыздандырудың жаңартулары осы Ubuntu нұсқасының "
"шығарылымынан кейін шығарылған. Егер де оларды қазір орнатуыңыз келіп тұрған "
"жоқ болса, кейінірек Әкімшілік мәзірінен \"Жаңарту менеджерін\" қолданып "
"орната аласыз."

#: ../UpdateManager/UpdateManager.py:692
msgid "Software updates are available for this computer"
msgstr ""
"Осы компьютерге бағдарламалық қамтамасыздандырудың жаңартулары жетімді"

#: ../UpdateManager/UpdateManager.py:694
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from "
"Applications later."
msgstr ""
"Егер де оларды қазір орнатуыңыз келіп тұрған жоқ болса, онда кейін оларды "
"Бағдарламалар мәзірінен \"Жаңартулар менеджері\" таңдау арқылы ала аласыз."

#: ../UpdateManager/UpdateManager.py:696
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Егер де сіз оларды қазір орнатпаймын десеңіз, кейінірек оны Жүйелік бас "
"мәзіріндегі Әкімшілік мәзірінде \"Жаңарту менеджерінде\" жасауыңыз мүмкін."

#: ../UpdateManager/UpdateManager.py:748
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Жаңарту жасау үшін \"%s\" дискісінде %s бос дискілік орын қажет. \"%s\" "
"дискісінде ең аз дегенде %s дискілік орын босатуыңыз керек. \"sudo apt-get "
"clean\" команданы орындау арқылы өзіңіздің корзинаңызды тазартып, "
"бағдарламалар орнатылымның уақытша пакеттерді жойыңыз."

#: ../UpdateManager/UpdateManager.py:773
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Жаңартуды толығымен орнату үшін, компьютерді қайта жүктеу қажет. Әрі қарай "
"жалғастыру алдында, өзіңіздің жұмысыңызды сақтаңыз."

#: ../UpdateManager/UpdateManager.py:837
msgid "Reading package information"
msgstr "Пакет туралы ақпаратты оқу"

#: ../UpdateManager/UpdateManager.py:852
msgid "Connecting..."
msgstr "Қосылуда..."

#: ../UpdateManager/UpdateManager.py:869
msgid "You may not be able to check for updates or download new updates."
msgstr "Жаңартуларға тексеру немесе жүктеуге құқықтар жетіспейді."

#: ../UpdateManager/UpdateManager.py:994
msgid "Could not initialize the package information"
msgstr "Пакеттер жайлы ақпаратты алу мүмкін емес"

#: ../UpdateManager/UpdateManager.py:995
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Пакет туралы ақпаратты ашу кезінде шешілмейтін қате туындады.\n"
"\n"
"\"update-manager\" пакетінің осы қате туралы хабарлап, оған қосы келесі "
"хабарламаны қосыңыз:\n"

#: ../UpdateManager/UpdateManager.py:1024
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Жаңартуды есептеу кезінде шешілмейтін қате туындады.\n"
"\n"
"\"update-manager\" пакетінің осы қате туралы хабарлап, оған қосы келесі "
"хабарламаны қосыңыз:"

#: ../UpdateManager/UpdateManager.py:1049
msgid " (New install)"
msgstr " (Жаңа орнатылым)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1056
#, python-format
msgid "(Size: %s)"
msgstr "(Мөлшер: %s)"

#: ../UpdateManager/UpdateManager.py:1058
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "%(old_version)s нұсқасынан %(new_version)s нұсқасына"

#: ../UpdateManager/UpdateManager.py:1062
#, python-format
msgid "Version %s"
msgstr "Нұсқа %s"

#: ../UpdateManager/UpdateManager.py:1093 ../do-release-upgrade:102
msgid "Release upgrade not possible right now"
msgstr "Дәл қазір релизді жаңартуы мүмкін емес"

#: ../UpdateManager/UpdateManager.py:1094 ../do-release-upgrade:103
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Релиздың жаңартуын қазір орындауға болмайды, кейінірек қайта жасап көріңіз. "
"Сервер жауабы: \"%s\""

#: ../UpdateManager/UpdateManager.py:1096 ../check-new-release-gtk:116
msgid "Downloading the release upgrade tool"
msgstr "Дистрибутивті жаңарту құралы жүктелуде"

#: ../UpdateManager/UpdateManager.py:1103
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Ubuntu жаңа \"%s\" релизі жетімді</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1142
msgid "Software index is broken"
msgstr "Бағдарламалардың индексі бұзылған"

#: ../UpdateManager/UpdateManager.py:1143
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Бағдарламаларды орнату немесе жою мүмкін емес. Мәселені шешу үшін "
"\"Synaptic\" пакеттер менеджерін қолданыңыз немесе терминалда \"sudo apt-get "
"install -f\" орындаңыз."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr "Жаңартулардың жетімдігіне тексеру"

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr "Барлық жетімді жаңартуларды орнату"

#: ../UpdateManagerText/UpdateManagerText.py:31
msgid "Cancel"
msgstr "Болдырмау"

#: ../UpdateManagerText/UpdateManagerText.py:32
msgid "Install"
msgstr "Орнату"

#: ../UpdateManagerText/UpdateManagerText.py:50
msgid "Building Updates List"
msgstr "Жаңартулар тізімі құрастырылуда"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Кәдімгі жаңарту есептеліне алынбай жатыр, осыны орындап көріңіз: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Келесідей пайда болу себептері болуы мүмкін:\n"
" * Алдыңғы аяқталмаған жаңарту кесірінен\n"
" * Кейбір орнатылған бағдарламалар немесе пакеттердің мәселелер кесірінен\n"
" * Ubuntu-да қолдау жоқ ресми емес пакеттердің кесірінен\n"
" * Ubuntu алдын ала нұсқасындағы жасалған өзгертулер кесірінен"

#: ../UpdateManagerText/UpdateManagerText.py:122
msgid "Downloading changelog"
msgstr "Өзгертулер тізімін жүктеу"

#: ../UpdateManager/Core/MyCache.py:128
#, python-format
msgid "Other updates (%s)"
msgstr "Басқа жаңартулар (%s)"

#: ../UpdateManager/Core/MyCache.py:283
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""
"Нұсқа өзгертулері:\n"
"Орнатылған нұсқа: %s\n"
"Жетімді нұсқа: %s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:297
msgid "This update does not come from a source that supports changelogs."
msgstr ""
"Бұл өзгерту өзгертулердің тарихын сақтамайтын қайнар көзінен ұсынылған."

#: ../UpdateManager/Core/MyCache.py:303 ../UpdateManager/Core/MyCache.py:324
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Өзгертулер тізімін жүктеу сәтсіз аяқталды. \n"
"Өзіңіздің интернет қосылысыңызды тексеріңіз."

#: ../UpdateManager/Core/MyCache.py:313
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Өзгертулер тізімі осы пакетпен байланысты жазбалары әзірше жоқ.\n"
"\n"
"Өзгертулер тізімі дайын болғанша "
"http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"қараңыз немесе өзгертулер тізімін кейінірек қайтадан байқап көріңіз."

#: ../UpdateManager/Core/MyCache.py:318
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Өзгертулер тізімі әзірше жоқ.\n"
"\n"
"Өзгертулер тізімі дайын болғанша "
"http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"қараңыз немесе өзгертулер тізімін кейінірек қайтадан байқап көріңіз."

#: ../UpdateManager/Core/UpdateList.py:49
msgid "Failed to detect distribution"
msgstr "Дистрибутивті анықтау мүмкін емес"

#: ../UpdateManager/Core/UpdateList.py:50
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr ""
"Қандай жүйенің қолданып жатқаныңызды анықтау кезінде \"%s\" қатесі туындады."

#: ../UpdateManager/Core/UpdateList.py:61
msgid "Important security updates"
msgstr "Маңызды қауіпсіздік жаңартулары"

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Recommended updates"
msgstr "Қолдануға ұсынылған жаңартулар"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Proposed updates"
msgstr "Ұсынылған жаңартулар"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Backports"
msgstr "Бұрыңғы нұсқаларынан бейімделу (backports-бэкпорты)"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Distribution updates"
msgstr "Дистрибутив жаңартулары"

#: ../UpdateManager/Core/UpdateList.py:70
msgid "Other updates"
msgstr "Басқа жаңартулар"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Жаңартуларды жетімдігін жекешелей (вручную) байқап көруіңіз "
"қажет</big></b>\n"
"\n"
"Сіздің жүкеңіз жаңартулардың жетімдігін тексермейді. Оны <i>Бағдарламалар "
"қайнар көздерінен</i>, <i>Жаңартулар</i> бетбелгісінде өзгертуіңізге болады."

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Кейбір жаңартуларды орнату мүмкін емес</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Батарейден жұмыс істелуде</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Жаңарту менеджері ашылуда</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:7
msgid "Changes"
msgstr "Өзгерістер"

#: ../data/gtkbuilder/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "_Тексеру"

#: ../data/gtkbuilder/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "_Жалғастыру"

#: ../data/gtkbuilder/UpdateManager.ui.h:10
msgid "Description"
msgstr "Сипаттама"

#: ../data/gtkbuilder/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "Жаңартудың сипаттамасы"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""
"Жаңарту алдында компьютерді электр қуат көзіне қосқаны қауіпсіздеу болады."

#: ../data/gtkbuilder/UpdateManager.ui.h:14
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Мүмкіндігінше көп жаңартулары бар, дистрибутив жарым-жарты жаңартуын "
"(частичное обновление) орындау. \n"
"\n"
"Мүмкін болатын себептер:\n"
"\n"
" * Алдыңғы аяқталмаған дистрибутив жаңарту кесірінен\n"
" * Қандай да бір орнатылған бағдарламалық қамтамасыздандыруымен бар "
"мәселелер кесірінен\n"
" * Ubuntu-да қолдау жоқ ресми емес пакеттердің кесірінен\n"
" * Ubuntu алдын ала нұсқасындағы жасалған өзгертулер кесірінен"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Show progress of individual files"
msgstr "Бөлек файлдарға барысын көрсету"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
msgid "Starting Update Manager"
msgstr "Жаңарту менеджері ашылуда"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
msgid "U_pgrade"
msgstr "_Жаңарту"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
#: ../data/update-manager.desktop.in.h:3
msgid "Update Manager"
msgstr "Жаңарту менеджері"

#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "You are connected via a wireless modem."
msgstr "Сіз сымсыз модем арқылы қосылып тұрсыз."

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Сіз роумингте қосылып тұрсыз, бұл жаңарту кезінде алынатын мәліметке қосымша "
"ақы төлеуіңізге әкелу мүмкін."

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr ""
"Сіздің компьютеріңіз батарейден жұмыс істеп жатыр. Әрі қарай жалғастырасыз "
"ба?"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "_Hide this information in the future"
msgstr "Осы мәліметтерді ендігәрі _көрсетпеу"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
msgid "_Install Updates"
msgstr "Жаңартуларды _орнату"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid "_Partial Upgrade"
msgstr "_Жарым-жартылай жаңарту"

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "_Settings..."
msgstr "_Баптаулар..."

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Upgrade"
msgstr "_Жаңарту"

#: ../data/gtkbuilder/UpdateManager.ui.h:35
msgid "updates"
msgstr "жаңартулар"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Ubuntu жаңа нұсқасы жетімді. Жаңартасыз ба?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Ask Me Later"
msgstr "Кейінірек сұрау"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Жаңартпау"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "Yes, Upgrade Now"
msgstr "Иә, қазір жаңарту"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid ""
"You can upgrade at a later time by opening Update Manager and click on "
"\"Upgrade\"."
msgstr ""
"Жаңарту менеджерін ашып, \"Жаңартуларды орнату\" батырмасын басу арқылы, "
"жүйені кейнірік жаңартуыңызға болады."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Сіз Ubuntu-ның келесі нұсқасына дейін жаңартуды кері қайтардыңыз"

#: ../data/update-manager.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Жетімді жаңартуларды көрсету және орнату"

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Бағдарламалардың жаңартулары"

#: ../update-manager:54 ../do-release-upgrade:41
msgid "Show version and exit"
msgstr "Үлгісін көрсету және шығу"

#: ../update-manager:57
msgid "Directory that contains the data files"
msgstr "Мәліммет файлдары бар бума"

#: ../update-manager:60
msgid "Check if a new Ubuntu release is available"
msgstr "Ubuntu-ның жаңа релизінің жетімдігін тексеру"

#: ../update-manager:63 ../do-release-upgrade:44 ../check-new-release-gtk:176
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Дистрибутивтің соңғы тұрақсыз үлгісіне дейін жаңарту мүмкіндігін тексеру"

#: ../update-manager:67
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "Жаңарту менеджерінің соңғы үлгісін қолданып жаңартуларды жасау"

#: ../update-manager:74
msgid "Do not focus on map when starting"
msgstr "Ашылатын кезде картаны фокустамау (таңдамау)"

#: ../update-manager:77
msgid "Try to run a dist-upgrade"
msgstr "dist-upgrade командасын орындап көріңіз"

#: ../update-manager:80
msgid "Do not check for updates when starting"
msgstr "Компьютер қосылған кезде жаңартуларға жетімдігін тексермеу"

#: ../update-manager:84 ../do-release-upgrade:60
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Жаңартуларды қауіпсіз режимінде тестілеу"

#: ../update-manager:104
msgid "Running partial upgrade"
msgstr "Жарым-жартылай жаңарту жасалынуда"

#: ../do-release-upgrade:48 ../check-new-release-gtk:180
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"$distro-proposed бұтағындағы жаңарту құралын қолданып, соңғы релизге дейін "
"жаңартып көріңіз"

#: ../do-release-upgrade:52
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Жаңартуды арнайы режимін ашу.\n"
"Қазіргі уақытта жүйелі жаңартулар \"***настольный***\" және \"серверлік\" "
"жүйелерге жетімді."

#: ../do-release-upgrade:58
msgid "Run the specified frontend"
msgstr "Көрсетілген интерфейсті ашу"

#: ../do-release-upgrade:63
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Дистрибутивтің жаңа нұсқасының жетімдігін тексеріп, нәтижесін шығу кодымен "
"шығару"

#: ../do-release-upgrade:77
msgid "Checking for a new ubuntu release"
msgstr "Ubuntu-ның жаңа релизінің жетімдігін тексеру"

#: ../do-release-upgrade:91
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Жаңарту жайлы мәліметті осы жерден таба аласыз:\n"
"%(url)s\n"

#: ../do-release-upgrade:97
msgid "No new release found"
msgstr "Жаңа релиз табылмады"

#: ../do-release-upgrade:109
#, c-format
msgid "New release '%s' available."
msgstr "\"%s\" жаңа релизі жетімді"

#: ../do-release-upgrade:110
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr ""
"Сол релизге дейін жаңартылу үшін \"do-release-upgrade\" командасын орындаңыз."

#: ../check-new-release-gtk:91
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Ubuntu %(version)s жаңартуы жетімді"

#: ../check-new-release-gtk:132
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Сіз Ubuntu-ды %s дейін жаңартуды кері қайтардыңыз"

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Орындалмаған әдіс: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Дисктегі файл"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb пакеті"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Жетіспеген пакетті орнату."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "%s пакетті орнату қажет."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "Күй файлдың %i құрамдасы ескірген"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "dpkg күйдің құрамы ескірген"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "dpkg күйлер құрамы ескірген"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Жаңарту кезінде, орнатылған kdelibs5-dev орнына kdelibs4-dev орнату қажет. "
"Бұл қате жайлы осы жерде оқуыңызға болады, bugs.launchpad.net, bug #279621."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s жекешелей орнатылған етіп белгілену қажет."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr "Егер grub орнатылған болса, lilo жойыңыз. (толығырақ, bug #314004)"

#, python-format
#~ msgid "%.0f KB"
#~ msgstr "%.0f Кб"

#~ msgid "1 KB"
#~ msgstr "1 Кб"

#~ msgid "0 KB"
#~ msgstr "0 Кб"

#~ msgid ""
#~ "It was impossible to install a required package. Please report this as a "
#~ "bug. "
#~ msgstr "Қажетті пакетті орнатуға мүмкін емес. Қате туралы есепті жіберіңіз. "

#~ msgid "The window size"
#~ msgstr "Терезенің өлшемі"

#~ msgid ""
#~ "If none of this applies, then please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Егер ештеңе көмектемесе, \"update-manager\" пакетімен байланысты қате туралы "
#~ "хабарлаңыз, және де оған қоса /var/log/dist-upgrade/ бумасындағы файлдарды "
#~ "хабарламаға тиеңіз."

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug."
#~ msgstr ""
#~ "Репозиторий туралы ақпарат жаңарту кезінде қате файл құрылған. Қате туралы "
#~ "хабарлаңыз."

#~ msgid ""
#~ "This usually is caused by a system were /tmp is mounted noexec. Please "
#~ "remount without noexec and run the upgrade again."
#~ msgstr ""
#~ "Әдетте бұл мәселе /tmp бумасы noexec баптауымен құрастырылған "
#~ "(смонтированный) жағдайда пайда болады. Ол буманы noexec баптауымен қайта "
#~ "құрастырып (смонтировать) жаңартуды жаңадан жасаңыз."

#~ msgid "_Check All"
#~ msgstr "Барлығын _белгілеу"

#~ msgid "_Uncheck All"
#~ msgstr "Барлық белгілерді _алып тастау"

#, python-format
#~ msgid ""
#~ "Changes for the versions:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"
#~ msgstr ""
#~ "Нұсқалардағы өзгертулер:\n"
#~ "%s\n"
#~ "%s\n"
#~ "\n"

#~ msgid "Previous selected"
#~ msgstr "Алдыңғы белгіленгендер"

#~ msgid "Settings..."
#~ msgstr "Баптаулар..."

#~ msgid "Show details of an update"
#~ msgstr "Жаңартуның толық ақпаратын көрсету"

#~ msgid "If this key is set a first run welcome message will be presented."
#~ msgstr ""
#~ "Егер де баптау таңдалынған болса, қарсы алу мәлімдемесі көрсетілетін болады."

#~ msgid "Remind to reload the channel list"
#~ msgstr "Қайнарлар тізімін жаңарту туралы ескерті"

#~ msgid "Show version in update list"
#~ msgstr "Жаңарту тізімінде нұсқаны көрсету"

#~ msgid "First run welcome message"
#~ msgstr "Бірінші ашылу кезіндегі шығатын қарсы алу мәлімдеме"

#~ msgid "Auto close the install window after successful install"
#~ msgstr "Сәтті орнатылымнан кейін терезені жабу"

#~ msgid ""
#~ "If this key is set the main update list window will show version information "
#~ "(from version to version)."
#~ msgstr ""
#~ "Егер де осы кілт орнатылған болса, басты терезесінің жаңарту тізімінде "
#~ "нұсқалар туралы ақпарат (нұсқадан нұсқаға) көрсетіледі."

#~ msgid ""
#~ "If this key is set the install window will be automatically closed on "
#~ "successful installation."
#~ msgstr ""
#~ "Егер белгі қойылған болса, орнатылым терезесі, орнатылымның сәтті аяқтауынан "
#~ "кейін жабылатын болады."

#~ msgid "Stores the size of the update-manager dialog"
#~ msgstr "Жаңарту менеджер терезесінің ауқымын сақтау"

#~ msgid ""
#~ "This is most likely a bug in the upgrade tool. Please report it as a bug"
#~ msgstr "Бұл жаңарту құралындағы қате бар сияқты. Осы қате туралы хабарлаңыз."

#~ msgid ""
#~ "It is unknown when the package information was updated last. Please try "
#~ "clicking on the 'Check' button to update the information."
#~ msgstr ""
#~ "Пакет туралы ақпарат қашан жаңартылғаны белгісіз. Ақпаратты жаңарту үшін "
#~ "\"Тексеру\" батырмасын басыңыз."

#~ msgid ""
#~ "If automatic checking for updates is disabled, you have to reload the "
#~ "channel list manually. This option allows to hide the reminder shown in this "
#~ "case."
#~ msgstr ""
#~ "Егер де жаңартуларды автоматты тексеру таңдалмаған болса, сіз қайнар көздер "
#~ "тізімін өзіңіз жаңартуыңыз қажет. Бұл баптау осы жағдайдағы шығаратын "
#~ "мәлімдемені жасырады."

#~ msgid ""
#~ "Stores the state of the expander that contains the list of changes and the "
#~ "description"
#~ msgstr "Сипаттамасы мен өзгертулер тізімі бар панельдің күйін сақтау"

#~ msgid "*DEPRECATED* this option will be ignore"
#~ msgstr "*ҚОЛДАНБАЙҒА ҰСЫНЫЛҒАН* бұл баптау қарастырылмайтын болады"

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this as a bug "
#~ "against the 'update-manager' package and include the files in /var/log/dist-"
#~ "upgrade/ in the bug report."
#~ msgstr ""
#~ "Жүйенің жаңартылуы сәтсіз аяқталды. \"update-manager\" пакеттің қатесі "
#~ "туралы хабарлаңыз, хабарламаға қоса /var/log/dist-upgrade/ бумасындағы "
#~ "файлдарды тиеңіз."

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The upgrade "
#~ "will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug against the 'update-manager' package and include "
#~ "the files in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Қажетті жаңарту шарттары орындалған жоқ. Жаңарту үзіліп, жүйе бастапқы күйге "
#~ "қайтарылатын болады.\n"
#~ "\n"
#~ "Осы қате туралы хабарлаңыз, хабарламаға қоса /var/log/dist-upgrade/ "
#~ "бумасындағы файлдарды тиеңіз."

#, python-format
#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Пакеттер жайлы мәліметтің жаңартуынан кейін, қажетті \"%s\" пакетін табу "
#~ "мүмкін емес болады.\n"
#~ "Бұл ауыр қате болып есептеледі. \"update-manager\" пакеттің қатесі туралы "
#~ "хабарлаңыз, хабарламаға қоса /var/log/dist-upgrade/ бумасындағы файлдарды "
#~ "тиеңіз."

#, python-format
#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bug report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "\"update-manager\" пакеттің қатесі туралы хабарлаңыз, хабарламаға қоса "
#~ "/var/log/dist-upgrade/ бумасындағы файлдарды тиеңіз.\n"
#~ "%s"

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration Menu later."
#~ msgstr ""
#~ "Осы бағдарламалық қамтамасыздандырудың жаңартулары Ubuntu релиздың "
#~ "шығарылғаннан кейін шығарылған. Оларды қазір орнатпасаңыз, кейінірек "
#~ "Әкімшілік мәзірінен \"Жаңарту менеджерін\" қолданып орната аласыз."

#~ msgid "This change is not coming from a source that supports changelogs."
#~ msgstr ""
#~ "Бұл өзгерту өзгертулердің тарихын сақтамайтын қайнар көзінен жасалған."

#~ msgid "make check-new-release-gtk ignore a given new release"
#~ msgstr "check-new-release-gtk орындалатын кезде, жаңа релиздерді қарастырмау"

#~ msgid ""
#~ "This will permanently hide the new release prompt from check-new-release-"
#~ "gtk. Note that the small button in the main update-manager UI will still be "
#~ "there."
#~ msgstr ""
#~ "check-new-release-gtk модуліне сұраным жасалатын кезде жаңа релиздердің "
#~ "жетімдігі туралы мәлімдеме көрсетілмейтін болады. Бірақ та Жаңарту менеджер "
#~ "бас терезесінде кішкене ғана батырма бұрындағыдай көріне беретіне назар "
#~ "салыңыз."

#~ msgid ""
#~ "This key is deprecated in favor of the file /etc/update-manager/release-"
#~ "upgrades Check automatically if a new version of the current distribution is "
#~ "available and offer to upgrade (if possible)."
#~ msgstr ""
#~ "/etc/update-manager/release-upgrades жаңартулардың жетімдігін автоматты "
#~ "түрде тексеріп, оларды орнатуға ұсынатындықтан (жетімді болатын болса) бұл "
#~ "кілт іске қосылмаған."

#~ msgid "*deprecated* Check for new distribution releases"
#~ msgstr ""
#~ "*қарастырылмайын* Дистрибутивтің жаңа шығарылымдарының (релиз) жетімдігін "
#~ "тексеру"

#~ msgid "Show package summary before package name in update list"
#~ msgstr "Жаңарту тізімінде пакет туралы ақпаратты, пакет атынан бұрын көрсету"

#~ msgid ""
#~ "The upgrade has aborted. Please check your Internet connection or "
#~ "installation media and try again. All files downloaded so far are kept."
#~ msgstr ""
#~ "Жаңарту үзілді. Интернетпен қосылысыңызды не орнатушы дискісін тексеріңіз "
#~ "де, жаңартуды қайтадан орындап көріңіз. Барлық жүктелген файлдар "
#~ "сақталынатын болады."

#~ msgid ""
#~ "If this key is set, the main update list window will show the package "
#~ "summary before the package name."
#~ msgstr ""
#~ "Егер бұл баптау қосылған болса, жаңарту тізімінде пакет туралы ақпаратты, "
#~ "пакет атынан бұрын көрсетілетін болады."

#~ msgid "The package information was last updated less than one hour ago."
#~ msgstr "Пакеттер жайлы мәлімет бір сағаттай уақыт аралығында жаңартылған."

#~ msgid "<b><big>Upgrading Ubuntu to version 11.04</big></b>"
#~ msgstr "<b><big>Ubuntu 11.04 нұсқасына дейін жаңарту</big></b>"

#~ msgid ""
#~ "These software updates have been issued since Ubuntu was released. If you "
#~ "don't want to install them now, choose \"Update Manager\" from Applications "
#~ "later."
#~ msgstr ""
#~ "Бұл бағдарламалардың жаңартулары Ubuntu шығарылымнан кейін шықты. Егер де "
#~ "оларды қазір орнатуыңыз келіп тұрған жоқ болса, онда кейін оларды "
#~ "Бағдарламалар мәзірінен \"Жаңартулар менеджері\" таңдау арқылы ала аласыз."
